packages/rum-core/test/common/event-target-patch.spec.js

```diff
@@ -0,0 +1,164 @@
+/**
+ * MIT License
+ *
+ * Copyright (c) 2017-present, Elasticsearch BV
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ */
+
+import patchEventHandler from './patch'
+import { EVENT_TARGET } from '../../src/common/constants'
+
+describe('EventTargetPatch', function() {
+  let events = []
+  let cancelFn
+
+  beforeAll(function() {
+    cancelFn = patchEventHandler.observe(EVENT_TARGET, function(event, task) {
+      events.push({
+        event,
+        task
+      })
+    })
+  })
+
+  afterAll(function() {
+    cancelFn()
+  })
+
+  beforeEach(function() {
+    events = []
+  })
+
+  function createEvent(eventType) {
```

Comment:
we have a similar function to create events. https://github.com/elastic/apm-agent-rum-js/blob/master/packages/rum-core/test/error-logging/error-logging.spec.js#L317-L331

Can we make use of that and move it to util?